var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -120,7 +120,6 @@ class Dealii(CMakePackage):
     def cmake_args(self):
         spec = self.spec
         options = []
-        options.extend(std_cmake_args)
 
         # CMAKE_BUILD_TYPE should be DebugRelease | Debug | Release
         for word in options[:]:
```

Comment:
@alalazo was not there a place to define a custom build type? I think there is...